release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -1927,6 +1927,9 @@ must prefix any custom MachineConfig file with a differing priority of `98-`
 instead of `99-`.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1826150[*BZ#1826150*])
 
+* If you do not specify an IPAM configuration in the SriovNetwork configuration, you must include an empty `ipam: {}` parameter or the NetworkAttachmentDefinition will not be created.
```

Comment:
I've been trying to follow this [Release Notes guide](https://mojo.redhat.com/docs/DOC-1216602) for writing known issues:

>Known Issues

>Use present tense for the issue and the imperative form for the workaround in the following format:

>There is currently a known issue <present tense explanation> (OPTIONAL: under the following <X conditions>).  Workaround: <workaround>.

With that in mind, perhaps we should split the problem/workaround into two separate sentences (avoiding past tense)? Perhaps something like this?

```
If you do not specify an IPAM configuration in the SriovNetwork configuration, the NetworkAttachmentDefinition is not created. As a workaround, you must include an empty `ipam: {}` parameter.
```

Reply:
Good point. The original is more minimalist, but this suggestion is probably easier to translate.

Reply:
Works for me - updated as suggested.
Thank you for the eyes @jeana-redhat and @codyhoag !